tests/ui/features/@vulnerability-explorer/vulnerability-explorer.feature

```diff
@@ -2,6 +2,21 @@ Feature: Vulnerability Explorer - View Vulnerability details
     Background: Authentication
         Given User is authenticated
 
+    # Search for vulnerability
+
+    # Vulnerability Explorer
+    Scenario: Display an overview of a vulnerability
+        Given User visits Vulnerability details Page of "<vulnerabilityID>"
+        Then The page title is "<vulnerabilityID>"
+        Then The severity is "<severityDescription>" and the CVSS score is "<severityScore>"
+        Then The description begins with "<descriptionBeginsWith>"
+        Then The Reserved date is "<dateReserved>", the Published date is "<datePublished>" and Last modified date is "<dateLastModified>"
```

Comment:
I am getting a timeout error for the "Reserved date" validation. Please find the attached results.
[Uploading trace.zip…]()